### ISSUE TITLE
[bug] convert-library decimated my ebook library

### ISSUE THREAD

**Issue by talex004** [crocodilestick/Calibre-Web-Automated#69]
**Describe the bug**
As instructed, I ran `convert-library -k` to convert my ebook library to epub format.
Somehow the script had the permissions to delete, but not to write a copy of the originals. (Or maybe because `original-library` destination folder didn't exist to begin with?) 
As a result, it deleted many of the books from (a copy of) my library, without maintaining the originals nor writing the replacement epub file.

**To Reproduce**
N/A

**Expected behavior**
`convert-library` should NOT delete the original files if it cannot verify that the copy was created successfully, or if cannot verify that the resulting epub was converted successfully.

**Screenshots**

```
root@84c7756e7f07:/# convert-library -k
[convert-library]: (1/15)  Converting xxx.mobi...
cp: cannot create regular file '/config/original-library/xxx.mobi': No such file or directory
Upgrading database to version 26...
Cannot read from /config/original-library/xxx.mobi
[convert-library]: (2/15)  Converting xxx.mobi...
sh: 1: Syntax error: "(" unexpected
Cannot read from /config/original-library/xxx.mobi
[convert-library]: (3/15)  Converting xxx.mobi...
cp: cannot create regular file '/config/original-library/xxx.mobi': No such file or directory
Cannot read from /config/original-library/xxx.mobi
...
```

**Configuration(please complete the following information):**
 - OS: Fedora CoreOS 40.20240808.3.0
 - Hardware: Proxmox VM, running on amd64 hardware.
 - Version: V2.0.1

**Additional context**
I'm mounting the ebook library through a podman volume, from a NFS server running on TrueNAS. I'm yet to figure out how to setup the correct permissions for this.

**Comment by jmarmstrong1207** [comment]
This exactly was my concern with the scripts and I noted this before here in https://github.com/crocodilestick/Calibre-Web-Automated/issues/33. This must be a high priority

**Comment by jmarmstrong1207** [comment]
Could you check if your books are in the docker container, in directory `/config/original-library`?

**Comment by jmarmstrong1207** [comment]
This has been fixed in 48b039bb3e9de6d0ff77c4407b1854091e921f5e
